### PR TITLE
Adding ci-chat-bot service account kubeconfigs to BitWarden

### PR DESCRIPTION
**Pull request by bradmwilliams**

### CODE REVIEW

File: core-services/ci-secret-generator/_config.yaml [hunk 1]
```diff
@@ -87,3 +87,13 @@
   params:
     cluster:
     - app.ci
+- item_name: ci-chat-bot
+  attachments:
+  - cmd: oc --context $(cluster) sa create-kubeconfig --namespace ci $(service_account) | sed "s/$(service_account)/$(cluster)/g"
+    name: sa.$(service_account).$(cluster).config
+  params:
+    cluster:
+    - api.ci
```
Comment (alvaroaleman) [inline core-services/ci-secret-generator/_config.yaml:96]: Why api ci? I thought it only creates prowjobs/talk to app.ci?

Comment (bradmwilliams) [reply core-services/ci-secret-generator/_config.yaml]: Copy an paste error.  My bad!  Thanks!

File: core-services/ci-secret-generator/_config.yaml [hunk 2]
```diff
@@ -88,3 +88,12 @@
   params:
     cluster:
     - app.ci
+- item_name: ci-chat-bot
+  attachments:
+  - cmd: oc --context $(cluster) sa create-kubeconfig --namespace ci $(service_account) | sed "s/$(service_account)/$(cluster)/g"
+    name: sa.$(service_account).$(cluster).config
```
Comment (alvaroaleman) [inline core-services/ci-secret-generator/_config.yaml:94]: Wait, apparently I got this wrong so:

* The chat bot uses the in-cluster config to talk to app.ci (so no need to involve bitwarden there)
* And it has mounted an extra config to talk to api.ci - That one needs to go through bitwarden

Comment (alvaroaleman) [reply core-services/ci-secret-generator/_config.yaml:94]: sorry for the confusion

Comment (bradmwilliams) [reply core-services/ci-secret-generator/_config.yaml:94]: No problem!  Im glad you mentioned it.  We can probably just scrape this PR altogether then and just remove passing the kubeconfig.

Comment (bradmwilliams) [reply core-services/ci-secret-generator/_config.yaml:94]: Sorry for the noise!

Comment (bradmwilliams) [reply core-services/ci-secret-generator/_config.yaml:94]: /close

**Pull request closed without merging**